src/Notifier.cpp

```diff
@@ -52,8 +54,8 @@ bool Notifier::Initialize(const size_t handshakeCount) {
       gQueue.reset(new Shotgun);
       try {
          gQueue->Aim(GetNotifierQueueName());
-      } catch (std::exception& e) {
-         LOG(WARNING) << "Caught exception: " << e.what();
+      } catch (std::string& e) {
```

Comment:
guys i did not write this code...it's been like this way for ever. do you want me to change it to an exception?